cfgov/unprocessed/js/routes/consumer-tools/financial-well-being/index.js

```diff
@@ -0,0 +1,85 @@
+/* eslint no-global-assign: "off" */
+'use strict';
+
+var Analytics = require( '../../../modules/Analytics' );
```

Comment:
Is this path right? If I watch the console when I click on the radio buttons, I'm getting "Uncaught ReferenceError: Analytics is not defined".

Reply:
Hang on... I hadn't pulled your commits before trying to test this 😞

Reply:
Lol 😂